docs/frontend/hover-intent.md

```diff
@@ -0,0 +1,32 @@
+# hoverIntent Component
```

Comment:
This should start with big letter and also name of the file should be same as first title

Reply:
fixed